packages/datadog_flutter_plugin/android/src/main/kotlin/com/datadoghq/flutter/DatadogSdkPlugin.kt

```diff
@@ -232,7 +246,31 @@ class DatadogSdkPlugin : FlutterPlugin, MethodCallHandler {
         )
     }
 
-    fun updateTelemetryConfiguration(call: MethodCall) {
+    private fun initWebView(webViewIdentifier: Long, allowedHosts: List<String>) {
+        // Plugin only accepts a FlutterEngine and this is the only way
+        // I know how to get it.
+        @Suppress("DEPRECATION")
+        val engine = binding?.flutterEngine
+        if (engine != null) {
+            val webView = WebViewFlutterAndroidExternalApi.getWebView(
+                engine, webViewIdentifier
+            )
+            if (webView != null) {
+                if (!webView.settings.javaScriptEnabled) {
+                    Log.e(DATADOG_FLUTTER_TAG, JAVA_SCRIPT_NOT_ENABLED_WARNING_MESSAGE)
+                } else {
+                    val bridge = DatadogEventBridge(allowedHosts)
+                    webView.addJavascriptInterface(bridge, "DatadogEventBridge")
```

Comment:
you can simply use `DatadogEventBridge#setup`

```suggestion
                    DatadogEventBridge.setup(webView, allowedHosts)
```

Reply:
Setup doesn't take allowed hosts as a parameter? It pulls from the configuration only I think.

Reply:
True, my bad, in v1 it is like that. In v2 I changed this method to take allowed hosts.